BitwardenShared/UI/Vault/Views/VaultListItemRow/VaultListItemRowView.swift

```diff
@@ -73,7 +73,7 @@ struct VaultListItemRowView: View {
                                     .accessibilityIdentifier("CipherSubTitleLabel")
                             }
                         }
-                        .accessibilityElement(children: .combine)
```

Comment:
🤔 Why do you need to remove the combine element?

Reply:
The accessibilityID we assigned to the element was repeatedly displayed on Appium Inspector for the same element (instead of "CipherCell", it was "CipherCell-CipherCell").

As part of my investigation, I removed those calls to check how it is displayed but forgot to add them again. 
I know these updates might break accessibility, so my idea was to manually validate how VoiceOver behaves on this view after the update, but thankfully, you and Matt noticed it before my tests  

The removed line will be added on my next commit 💪